src/get-url.ts

```diff
@@ -1,30 +1,30 @@
 export default function getURL(
-  os: string,
-  arch: string,
-  extended: string,
-  version: string
+  system: {
+    os: string,
+    arch: string,
+  },
+  options: {
+    extended: string,
+    withdeploy: string,
+    version: string  
+  }
 ): string {
-  const extendedStr = (extended: string): string => {
-    if (extended === 'true') {
-      return 'extended_';
-    } else {
-      return '';
-      // } else {
-      //   throw new Error(`Invalid input (extended): ${extended}`);
-    }
-  };
+  let extendedStr = '';
+  if (options.extended === 'true') {
```

Comment:
In order to enable extended whenever `withdeploy` is set, can the small change be made to move the withdeployStr declaration and conditional above this line, and then change it to `if (options.extended === 'true' || withdeployStr) {`?

Reply:
is it ok now?